inference-engine/tests/functional/inference_engine/lp_transformations/transformations_after_split_transformation.cpp

```diff
@@ -0,0 +1,220 @@
+        return transformer;
+    }
+    if (name == "ReshapeTransformation") {
+        transformer.add<ReshapeTransformation, ngraph::opset1::Reshape>(params);
+        return transformer;
+    }
+    if (name == "SqueezeTransformation") {
+        transformer.add<SqueezeTransformation, ngraph::opset1::Squeeze>(params);
+        return transformer;
+    }
+    if (name == "StridedSliceTransformation") {
+        transformer.add<StridedSliceTransformation, ngraph::opset1::StridedSlice>(params);
+        return transformer;
+    }
+    if (name == "TransposeTransformation") {
+        transformer.add<TransposeTransformation, ngraph::opset1::Transpose>(params);
+        return transformer;
+    }
+    if (name == "UnsqueezeTransformation") {
+        transformer.add<UnsqueezeTransformation, ngraph::opset1::Unsqueeze>(params);
+        return transformer;
+    }
+    if (name == "FuseConvertTransformation") {
+        transformer.add<FuseConvertTransformation, ngraph::opset1::Multiply>(params);
+        return transformer;
+    }
+    if (name == "FuseSubtractToFakeQuantizeTransformation") {
+        transformer.add<FuseSubtractToFakeQuantizeTransformation, ngraph::opset1::Subtract>(params);
+        return transformer;
+    }
+    if (name == "FuseMultiplyToFakeQuantizeTransformation") {
+        transformer.add<FuseMultiplyToFakeQuantizeTransformation, ngraph::opset1::Multiply>(params);
+        return transformer;
+    }
+    if (name == "MultiplyToGroupConvolutionTransformation") {
+        transformer.add<MultiplyToGroupConvolutionTransformation, ngraph::opset1::Multiply>(params);
+        return transformer;
+    }
+    if (name == "SubtractMultiplyToMultiplyAddTransformation") {
+        transformer.add<SubtractMultiplyToMultiplyAddTransformation, ngraph::opset1::Multiply>(params);
+        return transformer;
+    }
+    throw std::runtime_error("unexpected transformation name");
+}
+
+class TransformationsAfterSplitTransformation : public LayerTransformation, public testing::WithParamInterface<std::string> {
+public:
+    void SetUp() override {
+        const auto layerName = GetParam();
+        function = ngraph::builder::subgraph::TransformationsAfterSplitFunction::get(layerName);
+        function->validate_nodes_and_infer_types();
+    }
+
+    static std::string getTestCaseName(testing::TestParamInfo<std::string> obj) {
+        const auto layerName = obj.param;
+        std::ostringstream result;
+
+        result << "additional_layer_name_" << layerName;
+        return result.str();
+    }
+
+protected:
+    std::shared_ptr<ngraph::Function> function;
+};
+
+TEST_P(TransformationsAfterSplitTransformation, Run) {
+    const std::string layerName = GetParam();
+    const auto params = LayerTransformation::createParamsU8I8();
+    SimpleLowPrecisionTransformer transformer = getTransformerWithTransformationByName(params, layerName);
+
+    ASSERT_NO_THROW(transformer.transform(function));
```

Comment:
let's discuss offline: `TransformationsAfterSplitFunction::get` create dequantizations operations & test operation on all Split branches. Has test to fail if something wrong with input ports in tested transformation?

Reply:
will be fixed later